# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/cr144-esbuild-non-0.25.1.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

From f9545ffc3a26f2754e8ead3ada63dccf47e0de96 Mon Sep 17 00:00:00 2001
From: LN Liberda <lauren@selfisekai.rocks>
Date: Thu, 5 Feb 2026 04:33:27 +0100
Subject: [PATCH] Allow any esbuild version

---
 node_modules/esbuild/lib/main.js | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/third_party/devtools-frontend/src/node_modules/esbuild/lib/main.js b/third_party/devtools-frontend/src/node_modules/esbuild/lib/main.js
index aabd5e9e3f..fb24c35315 100644
--- a/third_party/devtools-frontend/src/node_modules/esbuild/lib/main.js
+++ b/third_party/devtools-frontend/src/node_modules/esbuild/lib/main.js
@@ -662,10 +662,6 @@ function createChannel(streamIn) {
   let handleIncomingPacket = (bytes) => {
     if (isFirstPacket) {
       isFirstPacket = false;
-      let binaryVersion = String.fromCharCode(...bytes);
-      if (binaryVersion !== "0.25.1") {
-        throw new Error(`Cannot start service: Host version "${"0.25.1"}" does not match binary version ${quote(binaryVersion)}`);
-      }
       return;
     }
     let packet = decodePacket(bytes);
